pkg/cmd/admin/validate/master.go

```diff
@@ -0,0 +1,182 @@
+package validate
+
+import (
+	"errors"
+	"fmt"
+	"io"
+	"os"
+	"text/tabwriter"
+
+	"github.com/spf13/cobra"
+
+	cmdutil "k8s.io/kubernetes/pkg/kubectl/cmd/util"
+	"k8s.io/kubernetes/pkg/util/fielderrors"
+
+	configapilatest "github.com/openshift/origin/pkg/cmd/server/api/latest"
+	"github.com/openshift/origin/pkg/cmd/server/api/validation"
+	"github.com/openshift/origin/pkg/cmd/util/clientcmd"
+)
+
+const (
+	ValidateMasterConfigRecommendedName = "master-config"
+
+	validateMasterConfigLong = `
+Validate the configuration file for a master server.
+
+This command validates that a configuration file intended to be used for a master server is 
+created correctly. 
+`
+
+	validateMasterConfigExample = ` // Validate master server configuration file
+  $ %s openshift.local.config/master/master-config.yaml`
+)
+
+type ValidateMasterConfigOptions struct {
+	// MasterConfigFile is the location of the config file to be validated
+	MasterConfigFile string
+
+	// Out is the writer to write output to
+	Out io.Writer
+}
+
+// NewCommandValidateMasterConfig provides a CLI handler for the `validate all-in-one` command
+func NewCommandValidateMasterConfig(name, fullName string, f *clientcmd.Factory, out io.Writer) *cobra.Command {
```

Comment:
> Both are used in cmd/admin/validate/validate.go

Chase the refs.  How do you use this here?  It's passed along, sure, but not used.